solvers/sklearn.py

```diff
@@ -1,5 +1,5 @@
 import warnings
-
+import numpy as np
```

Comment:
This needs to go in the `safe_import_context`, in order to make it fast to import this file for instance for autocomplete.